python/tritonserver/__init__.py

```diff
@@ -23,3 +23,51 @@
 # OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+"""In process Python API for Triton Inference Server."""
+
+import tritonserver._c as _triton_bindings
+from tritonserver._api.wrapper import InferenceRequest as InferenceRequest
+from tritonserver._api.wrapper import InstanceGroupKind as InstanceGroupKind
+from tritonserver._api.wrapper import LogFormat as LogFormat
+from tritonserver._api.wrapper import Metric as Metric
+from tritonserver._api.wrapper import MetricFamily as MetricFamily
+from tritonserver._api.wrapper import MetricFormat as MetricFormat
+from tritonserver._api.wrapper import MetricKind as MetricKind
+from tritonserver._api.wrapper import Model as Model
+from tritonserver._api.wrapper import ModelBatchFlag as ModelBatchFlag
+from tritonserver._api.wrapper import ModelControlMode as ModelControlMode
+from tritonserver._api.wrapper import ModelIndexFlag as ModelIndexFlag
+from tritonserver._api.wrapper import ModelTxnPropertyFlag as ModelTxnPropertyFlag
+from tritonserver._api.wrapper import Options as Options
+from tritonserver._api.wrapper import RateLimitMode as RateLimitMode
+from tritonserver._api.wrapper import Server as Server
+from tritonserver._c import AlreadyExistsError as AlreadyExistsError
+from tritonserver._c import InternalError as InternalError
+from tritonserver._c import InvalidArgumentError as InvalidArgumentError
+from tritonserver._c import NotFoundError as NotFoundError
+from tritonserver._c import TritonError as TritonError
+from tritonserver._c import UnavailableError as UnavailableError
+from tritonserver._c import UnknownError as UnknownError
+from tritonserver._c import UnsupportedError as UnsupportedError
```

Comment:
What is the use of `as ...`?

Reply:
I found a reference that this is preferred way of explicitly exporting for type checkers, might only be for stub files - but will double check. 

https://peps.python.org/pep-0484/#stub-files
https://github.com/microsoft/pylance-release/issues/2574